README.md

```diff
@@ -93,14 +99,19 @@ This URL can be used in Session to join the group
 
 ### Step 6: Add moderator to room
 
-in the root directory run 
+in the root directory run
 
-```python3 -msogs --rooms ROOMNAME --add-moderators SESSIONID```
+```bash
+python3 -msogs --rooms ROOMNAME --add-moderators SESSIONID
```

Comment:
Should probably also use `--admin` here, and should retitle the section to "Add admin to a room"